test/e2e/specs/specs-playwright/wp-plans_purchases-spec.ts

```diff
@@ -0,0 +1,116 @@
+import {
+	DataHelper,
+	LoginFlow,
+	setupHooks,
+	SidebarComponent,
+	PlansPage,
+	IndividualPurchasePage,
+	CartCheckoutPage,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+/**
+ * Constants
+ */
+
+const userOnPremiumPlan = 'defaultUser';
+
+describe( DataHelper.createSuiteTitle( 'Plans: Purchases' ), function () {
+	let page: Page;
+	let plansPage: PlansPage;
+	let purchasesPage: IndividualPurchasePage;
+	let cartCheckoutPage: CartCheckoutPage;
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	describe( 'Initial navigation', function () {
+		it( 'Log In', async function () {
+			const loginFlow = new LoginFlow( page, userOnPremiumPlan );
+			await loginFlow.logIn();
+		} );
+
+		it( 'Navigate to Plans from sidebar', async function () {
+			const sidebarComponent = new SidebarComponent( page );
+			await sidebarComponent.gotoMenu( { item: 'Upgrades', subitem: 'Plans' } );
+		} );
+
+		it( 'Land on the "My Plan" tab', async function () {
+			plansPage = new PlansPage( page );
+			await plansPage.validateActiveNavigationTab( 'My Plan' );
+		} );
+
+		it( 'The Premium plan is listed as current plan on "My Plan" tab', async function () {
+			await plansPage.validatePlanInMyPlan( 'Premium' );
+		} );
+
+		it( 'Click on the "Plans" navigation tab', async function () {
+			await plansPage.clickNavigationTab( 'Plans' );
+			await plansPage.validateActiveNavigationTab( 'Plans' );
+		} );
+	} );
+
+	describe( 'Manage current plan', function () {
```

Comment:
Could you specify what plan is active here currently? (WordPress.com Premium), just for clarify when reading the output.